components/Alert/alert.css

```diff
@@ -0,0 +1,77 @@
+.alert {
+  width: 25rem;
+  margin: 20px 0;
+  padding: 20px;
+  position: relative;
+  border-radius: 5px;
+  font-size: 0.9rem;
+}
+
+.close {
+  width: 30px;
+  height: 30px;
+  font-size: 1.2rem;
+  position: absolute;
+  top: 1.2rem;
+  right: 1.5rem;
+  cursor: pointer;
+  border-radius: 50%;
+  text-align: center;
+  font-family: monospace;
```

Comment:
Not sure if this is intentional or temporary, but font-family can be set in typography.

Reply:
Thanks for pointing out. I'll remove it.